src/libraries/System.Reflection/tests/MethodInfoTests.cs

```diff
@@ -894,8 +894,26 @@ private static void SecondCall(MethodInfo mi)
             Assert.Contains("TestAssembly", asm.ToString());
         }
 
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/71095", TestRuntimes.Mono)]
+        private static unsafe void TestFunctionPointersWithFunctionPointerArgs()
+        {
+            void* fn = FunctionPointerMethods.GetFunctionPointer();
+            MethodInfo m;
+
+            m = GetMethod(typeof(FunctionPointerMethods), nameof(FunctionPointerMethods.CallFcnPtr_FP));
+            Assert.True((bool)m.Invoke(null, new object[] { (IntPtr)fn, 42 }));
+            Assert.False((bool)m.Invoke(null, new object[] { (IntPtr)fn, 41 }));
```

Comment:
Add test with UIntPtr as well? (I assume that both IntPtr and UIntPtr work - is that right?)

Reply:
Yep. Done